apps/server/src/services/project-service/projectFileUtils.ts

```diff
@@ -14,18 +14,18 @@ export function filterProjectFiles(files: Array<string>): Array<string> {
   });
 }
 
+/**
+ * Parses a project file and returns the JSON object
+ * @param filePath
+ * @returns
+ */
 export function parseProjectFile(filePath: string): object {
   if (!filePath.endsWith('.json')) {
     throw new Error('Invalid file type');
   }
 
   const rawdata = readFileSync(filePath, 'utf-8');
-  const uploadedJson = JSON.parse(rawdata);
+  const jsonData = JSON.parse(rawdata);
```

Comment:
I wonder if we will think many times that this needs to be caught, we need a comment here to mention that the try-catch is on the consumer. maybe add an @throws directive in JSDocs